config/environments/production.rb

```diff
@@ -111,6 +111,9 @@
 
   if (ENV["AWS_ACCESS_KEY_ID"] || ENV["BUCKETEER_AWS_ACCESS_KEY_ID"]).present?
     config.active_storage.service = :amazon
+  else
+    puts "WARNING! : We didn't find an active_storage.service configured so we're falling back to the local store, but it's A VERY BAD IDEA to rely on it in product unless you know what your'e doing."
```

Comment:
Nit: think you mean: "production"?

Reply:
I did, indeed! Thanks for the sharp eye.